lib/ClangImporter/ClangImporter.cpp

```diff
@@ -2889,3 +2889,14 @@ void ClangImporter::Implementation::dumpSwiftLookupTables() {
   llvm::errs() << "<<Bridging header lookup table>>\n";
   BridgingHeaderLookupTable->dump();
 }
+
+DeclName ClangImporter::
+analyzeImportedName(const clang::NamedDecl *ClangDecl,
+                    clang::DeclarationName PreferredName,
+                    version::Version version) {
+  ImportNameVersion Ver = ImportNameVersion::Swift4;
+  if (version.isVersion3())
+    Ver = ImportNameVersion::Swift3;
```

Comment:
If this *does* need to be a separate function, can this use the same function that the regular name import uses? (`importer::nameVersionFromOptions`)